build/LegacyPackages/14.0/Microsoft.Build/Microsoft.Build.nuspec

```diff
@@ -0,0 +1,37 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata minClientVersion="2.8.1">
+    <id>$id$</id>
+    <version>$version$</version>
+    <title>$id$</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=624683</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <description>This package contains the $id$ assembly which is used to create, edit, and evaluation MSBuild projects.</description>
+    <tags>MSBuild</tags>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <dependencies>
+      <group targetFramework="netstandard1.3">
+        <dependency id="Microsoft.Build.Framework" version="[$version$]" />
+        <dependency id="System.Console" version="4.0.0" />
+        <dependency id="System.Diagnostics.Debug" version="4.0.11" />
+        <dependency id="System.Globalization" version="4.0.11" />
+        <dependency id="System.Xml.XmlDocument" version="4.0.1" />
+      </group>
+      <group targetFramework="net45">
+        <dependency id="Microsoft.Build.Framework" version="[$version$]" />
+      </group>
+    </dependencies>
+    <frameworkAssemblies>
+      <frameworkAssembly assemblyName="System.Xml" targetFramework="net45" />
+    </frameworkAssemblies>
+  </metadata>
+  <files>
+    <file src="lib\net45\$id$.dll" target="lib\net45\$id$.dll" />
+    <file src="bin\$configuration$\net45\$id$.dll" target="ref\net45\$id$.dll" />
```

Comment:
Why is there a ref assembly for .NET 4.5? Won't that make it hard to use the full desktop surface area from a full-framework project, even though that's ok?